tests/test_phase4_integration.py

```
@@ -0,0 +1,293 @@
+    from core.workflows.workflow_base import WorkflowConfig, WorkflowBase
+
+    # Create a processing config
+    config = ProcessingConfig(
+        extraction_method="auto",
+        enable_ocr=False,
+        extract_images=True
+    )
+
+    # Verify config attributes
+    assert config.extraction_method == "auto"
+    assert config.enable_ocr == False
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not config.enable_ocr:` for false checks

Replace with `not config.enable_ocr`

(E712)